### PR TITLE
perf mon using XDP & minor fixes

### PR DESCRIPTION
**Pull request by hasansayani-xilinx**
1. Added performance monitoring using XDP. Tested for host side. New architecture under implementation. APIs used will not change (ref: PaulS)
2. Added a fix for correct resource labelling

### CODE REVIEW

**Comment by hasansayani-xilinx** [comment]
Hey guys (@sonals, @sarab96), please can we have this pulled in?
Thanks!

**Comment by sarab96** [comment]
@hcneema  Please merge this

File: src/xma/src/xmaapi/CMakeLists.txt [hunk 1]
```diff
@@ -38,6 +38,7 @@ target_link_libraries(xmaapi
   stdc++
   xml2
   yaml
+  xdp
```
Comment (stsoe) [inline src/xma/src/xmaapi/CMakeLists.txt]: I don't think it is a good idea to introduce a link time dependency on xdp.
Dependencies should be other way around, xdp can depend in xrt, but not vice versa.
Look at runtime loading and symbol look-up similar to what is done for OCL.

Comment (hasansayani-xilinx) [reply src/xma/src/xmaapi/CMakeLists.txt]: Hey Soren, this is change is part of XMA, and does not belong to core XRT. Do you suggest we do not do this for XMA as well?

**Comment by hasansayani-xilinx** [comment]
closing this pull request and will create a new one to clear out stale refs.

**Pull request closed without merging**